src/main/resources/interface-specification.yml

```diff
@@ -465,26 +667,39 @@ components:
       properties:
         id:
           type: string
-          format: uuid
```

Comment:
this has been removed because the client returns a string instead of a UUID. Since changing the client may affect other services, at the moment we used the same datatype coming from attribute registry.

Reply:
I see, but being this the interface towards our users, I think that it should be formally correct, despite the implementations that we have beneath

Reply:
ok, so do you advice to change attribute-registry and fix this later?

Reply:
I would fix this now and, if we are in a rush, fix later the type in the attribute-registry.
Just because this could be shared at any moment with other parties

Reply:
ok